web-console/1-homepage/catalog.md

```diff
@@ -36,9 +33,32 @@ On smaller viewport sizes, the catalog tabs should stack. ![hover](img/OpenShift
 - For secondary levels of categorization, there should again be an `Other` category if there are items to display (not shown in mockup).
 - Clicking on a subcategory will open a scrollable panel that contains the catalog items.
 
+### Responsive State
+![mobile-1](img/OpenShift-Next-Mobile-V2-scrolledup.png)
+
+- On smaller viewport sizes, the catalog tabs should stack with arrows to the right.
+- Deploy Image and Import YAML should be displayed below all primary categories and should have a different visual treatment from the tabs.
+
+![mobile-2](img/OpenShift-Next-Mobile-subcategories.png)
+
+- If subcategories exist, selecting the primary category will bring users to another screen to view subcategories as tiles.
+- Clicking back will bring users back to the home page with all primary tabs listed.
+
+![mobile-3](img/OpenShift-Next-Mobile-subcategoryselected-V2.png)
+
+- Selecting a subcategory tile will bring users to another screen to view catalog items by scrolling.
+
+![mobile-4](img/OpenShift-Next-Mobile-subcategorydropdown.png)
```

Comment:
Do you have anything else in mind you wanted to try? The arrow dropdown seemed similar to the small caret we use in the masthead to signify there are additional options under there.